content/posts/gvisor.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Using gVisor with Fission"
+date: 2021-02-10T18:09:02+05:30
+author: "Harsh Thakur"
+categories: ["Fission", "Serverless", "Security", "gVisor"]
+description: "Enabling event driven autoscaling with KEDA integration"
+type: "post"
+---
+
+# Introduction
+
+Have you ever run into a scenario where you had to run untrusted code? Containers are great from a performance perspective but they have a considerable access to the kernel which can be exploited. In order to have the security of VMs and speed of containers, projects like [gVisor](https://github.com/google/gvisor) and [kata containers](https://github.com/kata-containers/kata-containers) have risen. In this post, we'll take a look at gVisor provides an application kernel for containers. It provides a runtime which can be used by Kubernetes. 
+
+# Setup of cluster
+
+Google Kubernetes Engine provides the option of using gVisor out of the box. You need to have a GKE cluster running, after which you can add a new node pool on which gVisor is enabled. The new node pool should have :
```

Comment:
Can we at least add pointers to how to run this if not on GKE - as not everyone will have GKE

Reply:
Sure. I'll add a link to other installations supported by gVisor : https://gvisor.dev/docs/user_guide/quick_start/kubernetes/

Reply:
Done,  PTAL